src/plugins/intel_cpu/src/config.cpp

```diff
@@ -4,6 +4,10 @@
 
 #include "config.h"
 
+#include <algorithm>
+#include <map>
+#include <string>
```

Comment:
It is better to put external header files after internal header files.

Reply:
Updated.